exopy_hqc_legacy/tasks/tasks/instr/tune_IQ_mixer.py

```diff
@@ -0,0 +1,223 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2018 by ExopyHqcLegacy Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""Tasks to set the parameters of arbitrary waveform generators.
+
+"""
+import logging
+import numbers
+import numpy as np
+
+from atom.api import (Enum, Unicode, Value)
+
+from exopy.tasks.api import (InstrumentTask, validators)
+import exopy_hqc_legacy.instruments.drivers.dll.SA124B as sa
+
+SA_SWEEPING = 0x0
+SA_REAL_TIME = 0x1
+
+
+class TuneIQMixerTask(InstrumentTask):
```

Comment:
Completing the work of Zaki on multi-instruments task would be best, I can guide you through it there is not much left to do if I remember properly.